### DISCUSSION TITLE
左下角标签界面切换功能失效了，打开之后没有显示了。之前的版本应该还是有的

### DISCUSSION THREAD

**Discussion by MrLittleShark** [MuiseDestiny/zotero-style#267] [Q&A]
![Snipaste_2023-03-17_11-58-15](https://user-images.githubusercontent.com/112299157/225809360-efcbb86b-98d3-4978-98df-cec8e2afb3d4.jpg)

**Comment by MuiseDestiny** [comment]
猜不到哪里出了问题，请补充错误日志，什么情况下出现这种状况，是否勾选“显示文库所有标签”

**Reply by MrLittleShark** [reply]
![image](https://user-images.githubusercontent.com/112299157/225815638-811ca17a-7712-4d8f-a7df-7d5b235430e0.png)
之前这里是可以切换标签的展示模式的，现在没了。还有，有一个bug，就是安装了插件后，这些列不能随意拖动了，比如我想把附件列挪到最后面，但是挪不了。
![image](https://user-images.githubusercontent.com/112299157/225815793-3f620006-4231-494c-997b-99a0e96695d8.png)

**Comment by Andreerassel** [comment]
我也是这个问题，左下角部分标签不显示了，明明一个文件有多个标签，只显示一个类，还有4类不显示，请问如何解决？
![Uploading image.png…]()
